build.gradle

```diff
@@ -24,6 +24,7 @@ tasks.withType(JavaCompile) {
 }
 
 dependencies {
+    testCompile 'com.fasterxml.jackson.core:jackson-core:2.8.4'
```

Comment:
It should not be necessary to add this dependency explicitly. jackson-databind should already depend on it.